src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,26 @@
+package duke.tasks;
+
+public class Task {
+    private final String taskName;
+    private Boolean isDone;
+    private final String taskType;
+
+    public Task(String taskName, String taskType) {
+        this.taskName = taskName;
+        this.isDone = Boolean.FALSE;
```

Comment:
Kind of curious as to why you didn't set this line as:
```suggestion
        this.isDone = false;
```
But this is more nit-picky then anything.